apps/services/user-profile/src/app/v2/me-user-profile.controller.ts

```diff
@@ -138,4 +143,42 @@ export class MeUserProfileController {
       this.userProfileService.confirmNudge(user.nationalId, input.nudgeType),
     )
   }
+
+  @Get('/actor-profiles')
+  @Scopes(UserProfileScope.read)
```

Comment:
I'm wondering if we should limit company access to the actor profile get and patch.

Companies get the user profile read and write scope, but it doesn't make sense for companies (procuration holders) to check and manage these settings. So this screen won't be available for companies and we might wan't to limit the api access as well. We could just use the `ApiScope.internal` instead, as that is a scope that only signed in users will get, no delegation session will get it.